src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1754,8 +1758,15 @@ private void visitEnumType(String symbolName, String qualifiedName, EnumType typ
         emit("{");
         emitBreak();
         indent();
-        for (String elem : sorted(type.getElements())) {
+        Map<String, String> elements =
+            Streams.stream(node.getNext().children())
+                .collect(
+                    Collectors.toMap(
+                        Node::getString, n -> String.valueOf(n.getFirstChild().getDouble())));
+        for (String elem : sorted(elements.keySet())) {
```

Comment:
why sorted? Isn't it best to emit in original order?

Reply:
It was sorted before, so I just kept the same functionality, I can remove this if you want.

I *think* it was being sorted because the values came from a `Set` and maybe they were generated in different orders, I guess that's not needed anymore

Reply:
ah, right. Ok to keep it as it is then.